gammapy/modeling/models/prior.py

```diff
@@ -196,3 +196,38 @@ def _inverse_cdf(self, value):
         """Return inverse CDF for prior."""
         rv = uniform(self.min.value, self.max.value - self.min.value)
         return rv.ppf(value)
+
+
+class LogUniformPrior(Prior):
+    """LogUniform Prior.
+
+    Equivalent to a uniform prior on the log of the parameter
+
+    Parameters
+    ----------
+    min : float
+        Minimum value.
+        Cannot be zero or negative.
```

Comment:
You might also add a test into the code to be sure;)

Reply:
This is a general question for gammapy API in general. 
Should we check that the user inputs a number and not a string, should we check for non negative values, etc
But I'm happy to put a test here